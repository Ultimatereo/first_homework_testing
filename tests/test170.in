3
1.87e-07 3.14e-07 4.76e-07 2.64e-07
3.34e-07 2.64e-07 4.28e-07 5.8e-08
4.97e-07 4.2e-08 2.6e-07 5e-09
