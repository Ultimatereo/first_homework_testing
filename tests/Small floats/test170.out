-1.45216
-3.35251
3.33664
